esp32/modules/easyrtc.py

```diff
@@ -7,8 +7,11 @@
 import machine, time
 
 # Functions
-def string(date=False, time=True):
-    [year, month, mday, wday, hour, min, sec, usec] = machine.RTC().datetime()
+def string(print_date=False, print_time=True, timestamp = -1):
+    if timestamp<0:
+        [year, month, mday, wday, hour, minute, sec, usec] = machine.RTC().datetime()
+    else:
+        [year, month, mday, hour, minute, second, wday, yday] = time.localtime(timestamp)
```

Comment:
they are completely different functions with a completely different return value...
I agree that sec should be second. Will change that